Only in cdd+-077-fixed: cdd+_gmp_init.cc
diff -c -r cdd+-077/gmp_init.cc cdd+-077-fixed/gmp_init.cc
*** cdd+-077/gmp_init.cc	Mon Jan  6 18:07:09 2003
--- cdd+-077-fixed/gmp_init.cc	Thu Jul 20 18:08:44 2006
***************
*** 31,36 ****
--- 31,38 ----
  #include "gmp_init.h"
  
  gmp_alloc_init::gmp_alloc_init() {
+ #if 0
+   /* This stuff does not compile with GCC newer that 3.3. */
  #ifdef _POLYMAKE_STD_STL_ALLOC_H
     typedef pm::raw_alloc raw_alloc;
  #elif defined(__GNUC__)
***************
*** 46,49 ****
--- 48,52 ----
        mp_set_memory_functions (raw_alloc::allocate, raw_alloc::reallocate, raw_alloc::deallocate);
        done=true;
     }
+ #endif
  }
Only in cdd+-077-fixed: gmp_init.cc~
diff -c -r cdd+-077/setoper.C cdd+-077-fixed/setoper.C
*** cdd+-077/setoper.C	Sun Apr 20 20:05:02 2003
--- cdd+-077-fixed/setoper.C	Thu Jul 20 17:45:17 2006
***************
*** 54,60 ****
  	long i,forlim1;
  	
  	forlim1=set_blocks(len);
! 	(*setp)=new (unsigned long)[forlim1];
  	(*setp)[0]=(unsigned long) len;  /* size of the ground set */
  	// cout << " size of the set is set to " << (*setp)[0] << "\n";
  	for (i=1; i<forlim1; i++)
--- 54,60 ----
  	long i,forlim1;
  	
  	forlim1=set_blocks(len);
! 	(*setp)=new unsigned long [forlim1];
  	(*setp)[0]=(unsigned long) len;  /* size of the ground set */
  	// cout << " size of the set is set to " << (*setp)[0] << "\n";
  	for (i=1; i<forlim1; i++)
Only in cdd+-077-fixed: setoper.C~
